### DISCUSSION TITLE
How to reverse the effects of cryptosmite?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1051] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there a specific tutorial or can someone write one for me on ho to reverse the effects of cryptosmite and re-enroll a chromebook?

**Comment by CJBrandi** [comment]
I have no clue. Anyone else?

**Comment by AnOrangeCone** [comment, marked as answer]
bruh esc power refresh then in the corner it says enroll device just click it or use sh1mmers method on there site (im pretty sure its exactly what i just said)
